.circleci/config.yml

```diff
@@ -3,23 +3,42 @@ version: 2.1
 workflows:
   test:
     jobs:
-      - build-run-linux
+    - build-run-linux:
+        context: hello-world-demos
+
+  test-daily:
+    triggers:
+    - schedule:
+        cron: "0 6 * * *"
+        filters:
+          branches:
+            only: master
+    jobs:
+    - build-run-linux:
+        context: hello-world-demos
 
 # This CI build ensures that the demo both compiles and works correctly. For the runtime test,
-# we use an SDK key and flag key that are passed in via the CircleCI project configuration;
-# the flag is configured to return a true value.
+# we use an SDK key and flag key that are passed in via the CircleCI context; the flag is
+# configured to return a true value.
 
 jobs:
   build-run-linux:
     docker:
-      - image: circleci/golang:1.14
+      - image: cimg/go:1.14
     steps:
       - checkout
+      - run:
+          name: verify that SDK dependencies have not been pinned
+          command: |
+            if grep 'gopkg.in/launchdarkly/[^ ]* v' go.mod; then
+              echo "Undesirable specific version dependency! See go.mod for explanation."
+              exit 1
+            fi
```

Comment:
This is new and is specific to Go - see comment in `go.mod` for an explanation.

Reply:
For an example of what a failure looks like, see: https://app.circleci.com/pipelines/github/launchdarkly/hello-go/13/workflows/363ad702-01a7-4c76-88f5-af6a670e5907/jobs/35